clientApp/src/components/pageHeader/header.module.scss

```
@@ -0,0 +1,41 @@
+@import "../../styles/variables.scss";
```

```suggestion
@import "styles/mixins.scss";
```

а внутри mixins стоит положить variables.scss